config/swagger-lume.php

```diff
@@ -200,7 +200,7 @@
      */
     'constants' => [
         // 'SWAGGER_LUME_CONST_HOST' => env('SWAGGER_LUME_CONST_HOST', 'http://my-default-host.com'),
-        'API_DESCRIPTION' => <<<EOF
+        'API_DESCRIPTION' => str_replace("\r\n", "\n", <<<EOF
```

Comment:
This should do the replacement of macOS and Windows line endings I think, unless macOS is using LF now.

Reply:
Yes, so this file on Windows probably have `\r\n` end of line depending on your git settings. This is here to have the same generated file on all well known OS.

Reply:
I thought that on macOS the line ending is `\r`, that's why I'm saying that this bit should cover that case if macOS is still using `\r` line endings.

Reply:
If not, then this is fine, my bad 😸

Reply:
It uses `\n` for a long time. Classic Mac OS used to use CR but that quite old (1984 - 2002).